### PR TITLE
Refactor melee sheet events to melee sheet

### PR DESCRIPTION
**Pull request by CarlosFdez**
I did this before I found it was a core issue, but its still an improvement so here we go.

### CODE REVIEW

File: src/module/item/melee/sheet.ts [hunk 1]
```diff
@@ -13,6 +13,29 @@ export class MeleeSheetPF2e extends ItemSheetPF2e<MeleePF2e> {
...
     }
+
+    override activateListeners($html: JQuery<HTMLElement>): void {
+        super.activateListeners($html);
+
+        // Add Damage Roll
+        $html.find(".add-damage").on("click", (event) => {
+            event.preventDefault();
+            const newKey = randomID(20);
+            const newDamageRoll = { damage: "", damageType: "" };
```
Comment (stwlam) [inline src/module/item/melee/sheet.ts:24]: Maybe give this a default, like 1d4 bludgeoning

File: src/module/item/melee/sheet.ts [hunk 2]
```diff
@@ -13,6 +13,29 @@ export class MeleeSheetPF2e extends ItemSheetPF2e<MeleePF2e> {
...
         };
     }
+
+    override activateListeners($html: JQuery<HTMLElement>): void {
+        super.activateListeners($html);
+
+        // Add Damage Roll
+        $html.find(".add-damage").on("click", (event) => {
+            event.preventDefault();
+            const newKey = randomID(20);
```
Comment (stwlam) [inline src/module/item/melee/sheet.ts:23]:
```suggestion
            const newKey = randomID();
```

File: src/module/item/melee/sheet.ts (hunk 1 above)
Comment (CarlosFdez) [reply src/module/item/melee/sheet.ts:24]: I think that'll make it more work to enter data, cause now they'll have to delete it right?

Comment (stwlam) [reply src/module/item/melee/sheet.ts:24]: It'll be valid, at least :shrug:

**Pull request merged**